pom.xml

```diff
@@ -208,6 +211,25 @@
                     </configLocation>
                 </configuration>
             </plugin>
+            
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
+                <version>${findbugs-maven-plugin.version}</version>
+                <configuration>
+                    <xmlOutput>true</xmlOutput>
+                    <failOnError>${findbugs.failOnError}</failOnError>
```

Comment:
i've think it is to have ability to redefine with help of `-Dfindbugs.failOnError` ... if it works without this property - it should be inlined, as it doesn't encrease readability